variables.tf

```
@@ -290,3 +290,9 @@ variable "enable_config_recorder" {
   type        = bool
   default     = true
 }
+
+variable "check_ec2-imdsv2-check" {
+  description = "Enable IMDSv2 rule"
+  type        = bool
+  default     = true
```

Since AWS intends to support the v1 version of the instance metadata [indefinitely](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html) and the migration to v2 is non-trivial I think it makes sense to leave this default disabled.
```suggestion
  default     = false
```